service/src/main/kotlin/fi/espoo/evaka/messaging/message/MessageQueries.kt

```diff
@@ -272,6 +273,74 @@ fun Database.Read.getMessage(id: UUID): Message {
         .single()
 }
 
+fun Database.Read.getCitizenReceivers(accountId: UUID): List<MessageAccount> {
+    val params = mapOf(
+        "accountId" to accountId,
+    )
+
+    // language=SQL
+    val sql = """
+WITH placement_ids AS (
+    SELECT pl.id AS id
+    FROM guardian g
+    JOIN placement pl
+    ON g.child_id = pl.child_id
+    WHERE guardian_id = (
+        SELECT person_id AS id
+        FROM message_account
+        WHERE id = :accountId
+    )
+    AND Daterange(pl.start_date, pl.end_date, '[]') @> current_date
```

Comment:
probably ok for this PR, but I anticipate a need to contact at least the supervisor in advance some time before the placement starts